advanced1/processing.php

```diff
@@ -0,0 +1,163 @@
+<?php
+require 'apicall.php';
+$main = new APIcall('https://www.innoraft.com/jsonapi/node/services');
+$main->call_api();
+$mainResponse = $main->get_data();
+
+/**
+ * Class Field
+ * 
+ * Class of the major sections (services) for the index page.
+ */
+class Field {
+  
+  /**
+   * Stores the title of the field.
+   * 
+   * @var string $title
+   */
+  public $title;
+  
+  /**
+   * Stores the id attribute of the field.
+   * 
+   * @var string $id
+   */
+  public $id;
+  
+  /**
+   * Stores the title of the field in HTML format complete with classes.
+   * 
+   * @var string $displayTitle
+   */
+  public $displayTitle;
+  
+  /**
+   * Stores a list of sub-field links in HTML format.
+   * 
+   * @var string $fieldServices
+   */
+  public $fieldServices;
+  
+  /**
+   * Stores the image link of the field object.
+   * 
+   * @var string $image
+   */
+  public $image;
+  
+  /**
+   * Stores the link of "Explore More" button.
+   * 
+   * @var string $link
+   */
+  public $link;
+  
+  /**
+   * Stores an array of the icons present in the field object.
+   * 
+   * @var array $link
+   */
+  public $icons = [];
+
+  /**
+   * Field constructor function.
+   *
+   * @param string $title The title of the field object.
+   * @param string $id  The id of the field object.
+   */
+  function __construct($title, $id) {
+    $this->title = $title;
+    $this->id = $id;
+  }
+
+  /**
+   * Function to get the image property of the object.
+   * 
+   * @return string
+   */
+  function get_image() {
+    return $this->image;
+  }
+
+  /**
+   * Function to get the title of the object in HTML format. 
+   *
+   * @return string
+   */
+  function get_displaytitle() {
+    return $this->displayTitle;
+  }
+
+  /**
+   * Function to get the sub-field links of the object in HTML format.  
+   *
+   * @return string
+   */
+  function get_services() {
+    return $this->fieldServices;
+  }
+  
+  /**
+   * Function to get the link for the "Explore More" button. 
+   *
+   * @return void
+   */
+  function get_link() {
+    return $this->link;
+  }
+}
+
+$objArr = [];
+for ($i = 0; $i < count($mainResponse['data']); $i++) {
+  $dataAttr = $mainResponse['data'][$i]['attributes'];
+  $dataRel = $mainResponse['data'][$i]['relationships'];
+  $field_services = $dataAttr['field_services']['value'];
+
+  if ($field_services !== NULL && $i > 11) {
+
+    $title = $dataAttr['title'];
+    // Performing another API call for getting the field object image.
+    $image = $dataRel['field_image']['links']['related']['href'];
+    $fieldImg = new APIcall($image);
+    $fieldImg->call_api();
+    $imageData = $fieldImg->get_data();
+    $imageLink = 'https://www.innoraft.com' . $imageData['data']['attributes']['uri']['url'];
+    // Storing the link for the "Explore More" button.
+    $link = 'https://www.innoraft.com' . $dataAttr['path']['alias'];
```

Comment:
Can't we add this link `https://www.innoraft.com` to a separate variable and use it here?